xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -473,6 +514,83 @@ mod tests {
 		assert_eq!(inverted, Err(()));
 	}
 
+	#[test]
+	fn global_consensus_converts_for_works() {
+		parameter_types! {
+			pub UniversalLocationInNetworkAbc: InteriorMultiLocation = X2(GlobalConsensus(ByGenesis([9; 32])), Parachain(1234));
+		}
+		let network_abc =
```

Comment:
Just my personal opinion, but imo `network_1, network_2, network_3` will make code easier to understand than `network_abc, network_xyz, network_efg` :)

Reply:
fixed